src/dags/stroomstoringen.py

```diff
@@ -0,0 +1,137 @@
+from datetime import datetime
+
+from airflow import DAG
+from airflow.models import Variable
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.postgres_operator import PostgresOperator
+from common import (
+    DATAPUNT_ENVIRONMENT,
+    SHARED_DIR,
+    MessageOperator,
+    default_args,
+    quote_string,
+    slack_webhook_token,
+)
+from common.db import DatabaseEngine
+from contact_point.callbacks import get_contact_point_on_failure_callback
+from http_fetch_operator import HttpFetchOperator
+from ogr2ogr_operator import Ogr2OgrOperator
+from postgres_permissions_operator import PostgresPermissionsOperator
+from postgres_rename_operator import PostgresTableRenameOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from sql.stroomstoringen import CONVERT_DATE_TIME
+
+dag_id: str = "stroomstoringen"
+variables: dict = Variable.get("stroomstoringen", deserialize_json=True)
+endpoint_url: str = variables["data_endpoints"]["stroomstoringen"]
+
+
+today = datetime.now().strftime("%m-%d-%Y")
```

Comment:
You should probably take the timezone into account. Even when you are only after the date, you don't know before hand when a DAG is going to be scheduled. You will get an incorrect result if the DAG is scheduled during an daylight savings change. And even if you do know it will not be issue in this case, within AIrflow we generally do take timezones into account. Given the fact that code gets copied it is best to do it properly.

Airflow uses the `pendulum` library. I'm not sure if pendulum timezones are compatible with those of the Python stdlib. Hence it is best to use `pendulum` exclusively

```python
import pendulum
from airflow.settings import TIMEZONE

# uppercase as it is used as a constant
TODAY = pendulum.now(TIMEZONE).format("MM-DD-YYYY")
```

BTW is the date format really month, day, year in that order?

Reply:
:+1: excellent 
Yes, its month, day, year (in that order). An American format apparently.